src/android/CordovaAdalPlugin.java

```diff
@@ -231,7 +231,7 @@ private boolean deleteTokenCacheItem(String authority, boolean validateAuthority
             return true;
         }
 
-        String key = CacheKey.createCacheKey(itemAuthority, resource, clientId, isMultipleResourceRefreshToken, userId);
+        String key = CacheKey.createCacheKey(itemAuthority, resource, clientId, isMultipleResourceRefreshToken, userId, null);
```

Comment:
@khalidaw Is it possible that a token cache item has `familyClientId` which is not null?
`deleteTokenCacheItem` will not work in this case.